doc/index.rst

```diff
@@ -67,6 +67,7 @@ Core Lightning Documentation
    lightning-invoice <lightning-invoice.7.md>
    lightning-invoicerequest <lightning-invoicerequest.7.md>
    lightning-keysend <lightning-keysend.7.md>
+   lightning-listaddresses <lightning-listaddresses.7.md>
```

Comment:
fwiw looks like this addition is causing problems w/ the CI build

```
+++ b/doc/index.rst
@@ -67,7 +67,6 @@ Core Lightning Documentation
    lightning-invoice <lightning-invoice.7.md>
    lightning-invoicerequest <lightning-invoicerequest.7.md>
    lightning-keysend <lightning-keysend.7.md>
-   lightning-listaddresses <lightning-listaddresses.7.md>
    lightning-listchannels <lightning-listchannels.7.md>
    lightning-listclosedchannels <lightning-listclosedchannels.7.md>
    lightning-listconfigs <lightning-listconfigs.7.md>
make: *** [Makefile:603: check-gen-updated] Error 1
```

Reply:
Accidental addition in my tree, fixed...